axelor-account/src/main/java/com/axelor/apps/account/service/batch/BatchReimbursementImport.java

```diff
@@ -97,12 +97,18 @@ protected void process() {
       Map<List<String[]>, String> data = null;
 
       try {
-        rejectImportService.createFilePath(accountConfig.getReimbursementImportFolderPathCFONB());
+
+        String dataImportDir = appBaseService.getFileUploadDir();
+
+        String reimbursementImportFolderPathCFONB =
+            dataImportDir + accountConfig.getReimbursementImportFolderPathCFONB();
+
+        rejectImportService.createFilePath(reimbursementImportFolderPathCFONB);
```

Comment:
Pass null value in parameter if accountConfig.getReimbursementImportFolderPathCFONB() is null